libtenzir/builtins/formats/json.cpp

```diff
@@ -1321,6 +1321,23 @@ class write_json_plugin final : public virtual operator_plugin2<write_json> {
   }
 };
 
+class write_ndjson_plugin final : public virtual operator_plugin2<write_json> {
+public:
+  auto name() const -> std::string override {
+    return "write_ndjson";
```

Comment:
With this, should we also remove the `ndjson=<bool>` option from `write_json`? That'd mirror the `read_json` and `read_ndjson` operators.